packages/tools/src/tools/ZoomTool.ts

```diff
@@ -12,7 +11,7 @@ import { EventTypes, PublicToolProps, ToolProps } from '../types';
  */
 class ZoomTool extends BaseTool {
   static toolName;
-  touchDragCallback: (evt: EventTypes.MouseDragEventType) => void;
+  touchDragCallback: (evt: EventTypes.TouchDragEventType) => void;
```

Comment:
This is a place where it does need to distinguish between touch drag and mouse drag - if it can automatically get the touch drag callback here, that would be really good.